main.js

```diff
@@ -0,0 +1,31 @@
+var $phrase = document.querySelector('#exercise1')
+
+var phrase = '"Life is short. Smile while you still have teeth."'
+var chars = phrase.split('')
+
+for (var i = 0; i < chars.length; i++) {
+  var $char = document.createElement('span')
+  $char.textContent = chars[i]
+  $phrase.appendChild($char)
+}
+
+$phrase.querySelector('span').classList.add('current')
+var $currentChar = document.querySelector('.current')
+
+
+
+/*
```

Comment:
Be sure not to turn in commented out code. Please delete these. You can always look at old commits, that's what git is for 😄